contributing.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Open Source
+
+Contributing to Open Source allows people to be able to give back and help
+contribute to projects that they use frequenty. Whether adding a feature
+to a project, fixing a bug, or adding documentation, it allows communities
+to form out of these projects.
+
+## Where to find projects
+
+Many widely known projects such as Firefox and VLC are open sourced and available 
```

Comment:
s/available to be contributed by members of the community/welcome contributions from newcomers./